pkg/gcloud/gcloud_test.go

```diff
@@ -67,6 +67,14 @@ func TestGetInstances(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
What are you trying to test here? How is this different from Instance 2?

Reply:
My attempt here was to simulate another instance but no NatIP